src/chart/sankey/SankeyView.ts

```diff
@@ -346,6 +363,70 @@ class SankeyView extends ChartView {
     }
 
     dispose() {
+        this._controller && this._controller.dispose();
+        this._controllerHost = null;
+    }
+
+    private _updateController(
```

Comment:
Overall, the code looks good and it works properly. It would be better with a small optimization: The code in this method is essentially the same as in TreeSeries, so could it be shared?